stylesheets/layout/footer.css

```diff
@@ -0,0 +1,80 @@
+/* footer css */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;

```

Comment:
You can extract this into a utility class.

Reply:
Added